### ISSUE TITLE
[abseil] build failure

### ISSUE THREAD

**Issue by fjcheng** [microsoft/vcpkg#40037]
### Operating system

win11

### Compiler

Mingw

### Steps to reproduce the behavior

```Shell
vcpkg install opencv:x64-mingw-dynamic
```

### Failure logs

CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: D:/dev/vcpkg/downloads/tools/cmake-3.29.2-windows/cmake-3.29.2-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j21
    Working Directory: D:/dev/vcpkg/buildtrees/abseil/x64-mingw-dynamic-dbg
    See logs for more information:
      D:\dev\vcpkg\buildtrees\abseil\install-x64-mingw-dynamic-dbg-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/abseil/portfile.cmake:34 (vcpkg_cmake_install)
  scripts/ports.cmake:192 (include)

error: building abseil:x64-mingw-dynamic failed with: BUILD_FAILED
Elapsed time to handle abseil:x64-mingw-dynamic: 8.7 s
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+abseil
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?title=[abseil]+Build+error+on+x64-mingw-dynamic&body=Copy+issue+body+from+D%3A%2Fdev%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md

[install-x64-mingw-dynamic-dbg-out.log](https://github.com/user-attachments/files/16342759/install-x64-mingw-dynamic-dbg-out.log)

### Additional context

I have already tried the command git pull and vcpkg update , but still have this issue.

**Comment by sfhacker** [comment]
This is NOT working!

**Comment by fjcheng** [comment]
> This is NOT working!

What do you mean? Did I install the instructions incorrectly, or was there a problem with the installation of this library？

**Comment by apollomok** [comment]
did you clone vcpkg from the github (git clone https://github.com/microsoft/vcpkg.git)?
run the bootstrap-vcpkg.bat
and then setup the environment variables?

For your reference:
https://learn.microsoft.com/en-us/vcpkg/get_started/get-started-vs?pivots=shell-cmd

**Comment by fjcheng** [comment]
> did you clone vcpkg from the github (git clone https://github.com/microsoft/vcpkg.git)? run the bootstrap-vcpkg.bat and then setup the environment variables?
> 
> For your reference: https://learn.microsoft.com/en-us/vcpkg/get_started/get-started-vs?pivots=shell-cmd

yes, I have installed fmt according to reference [](https://learn.microsoft.com/en-us/vcpkg/get_started/get-started-vs?pivots=shell-cmd).

<details> 
    <summary>my installed packages </summary>
   <pre><code>
PS C:\Users\XXX> vcpkg list
brotli:x64-mingw-dynamic                          1.1.0#1             a generic-purpose lossless compression algorithm...
brotli:x64-windows-release                        1.1.0#1             a generic-purpose lossless compression algorithm...
egl-registry:x64-mingw-dynamic                    2024-01-25          EGL API and Extension Registry
egl-registry:x64-mingw-static                     2024-01-25          EGL API and Extension Registry
eigen3:x64-mingw-static                           3.4.0#4             C++ template library for linear algebra: matrice...
fmt:x64-mingw-dynamic                             10.2.1#2            {fmt} is an open-source formatting library provi...
fmt:x64-mingw-static                              10.2.1#2            {fmt} is an open-source formatting library provi...
glad:x64-mingw-dynamic                            0.1.36              Multi-Language Vulkan/GL/GLES/EGL/GLX/WGL Loader...
glad:x64-mingw-static                             0.1.36              Multi-Language Vulkan/GL/GLES/EGL/GLX/WGL Loader...
glad[loader]:x64-mingw-dynamic                                        Generate loader logic.
glad[loader]:x64-mingw-static                                         Generate loader logic.
glfw3:x64-mingw-dynamic                           3.4                 GLFW is a free, Open Source, multi-platform libr...
glm:x64-mingw-dynamic                             1.0.1#3             OpenGL Mathematics (GLM)
imgui:x64-mingw-dynamic                           1.90.7#1            Bloat-free Immediate Mode Graphical User interfa...
imgui:x64-windows                                 1.90.7#1            Bloat-free Immediate Mode Graphical User interfa...
opengl-registry:x64-mingw-dynamic                 2024-02-10#1        OpenGL, OpenGL ES, and OpenGL ES-SC API and Exte...
opengl-registry:x64-mingw-static                  2024-02-10#1        OpenGL, OpenGL ES, and OpenGL ES-SC API and Exte...
opengl:x64-mingw-dynamic                          2022-12-04#3        Open Graphics Library (OpenGL)[3][4][5] is a cro...
opengl:x64-mingw-static                           2022-12-04#3        Open Graphics Library (OpenGL)[3][4][5] is a cro...
vcpkg-cmake-config:x64-windows                    2024-05-23
vcpkg-cmake:x64-windows                           2024-04-23
zlib:x64-mingw-dynamic                            1.3.1               A compression library
zlib:x64-mingw-static                             1.3.1               A compression library
zlib:x64-windows                                  1.3.1               A compression library
</code></pre>
</details>

<details> 
    <summary>my environment variables</summary>
   <pre><code>
PS C:\Users\XXX> $env:VCPKG_ROOT
D:\dev\vcpkg

PS C:\Users\XXX> $env:path
D:\VMware\VMware Workstation\bin\;C:\Windows\system32;C:\Program Files\mingw64\bin;D:\dev\vcpkg;D:\dev\boost_1_71_0\lib64-msvc-14.2;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.2\bin;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.2\libnvvp;%QT_DIR%\msvc2017_64\bin;C:\Program Files\Common Files\Oracle\Java\javapath;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\nodejs;C:\Program Files\Git\cmd;C:\Program Files\Graphviz\bin;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\Program Files\Microsoft SQL Server\120\Tools\Binn\;D:\Program Files\OCCT;D:\Program Files\OCCT\win64\vc14\bind;D:\Program Files\OCCT\products\tcltk-86-64\bin;D:\Program Files\OCCT\products\freetype-2.5.5-vc14-64\bin;D:\Program Files\OCCT\products\freeimage-3.17.0-vc14-64\bin;C:\Program Files\NVIDIA Corporation\Nsight Compute 2023.2.0\;D:\HEEDS\MDO\Ver2210.0001\Win64\;C:\Program Files (x86)\ZBar\bin;D:\Program Files (x86)\Graphviz\bin;C:\Program Files\dotnet\;d:\Program Files\010 Editor;D:\Program Files\nodejs\;D:\dev\glfw-3.4.bin.WIN64\lib-vc2019;C:\Program Files\Microsoft SQL Server\150\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;D:\dev\strawberry\c\bin;D:\dev\ninja-win;D:\Program_Files\anaconda3;D:\Program_Files\anaconda3\Scripts;D:\Program_Files\anaconda3\Library\bin;C:\Program Files\CMake\bin;C:\Program Files\MySQL\MySQL Shell 8.0\bin\;C:\Users\XXX\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Microsoft VS Code\bin;C:\Users\XXX\AppData\Roaming\npm
</code></pre>
</details>

**Comment by dg0yt** [comment]
You probably face a race with your antivirus software.

> `The process cannot access the file because it is being used by another process.`

**Comment by KaitechWord** [comment]
I have exactly same issue

**Comment by fjcheng** [comment]
After I reinstalled the OS, I was able to install it successfully but not use it. Eventually I switched to msys2's UCRT64.
